src/Core/Checkout/Cart/Rule/LineItemPurchasePriceGrossRule.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Checkout\Cart\Rule;
+
+use Shopware\Core\Checkout\Cart\LineItem\LineItem;
+
+class LineItemPurchasePriceGrossRule extends AbstractLineItemPurchasePriceRule
```

Comment:
Please merge the gross and net rule to a single pruchase price rule with a parameter for the net or gross value.

Reply:
That's a good suggestion. I added a small "select gross/net" field to the rule builder administration components. That way it is also easier to use for the administration user:
<img width="801" alt="Screenshot 2020-04-02 at 09 49 48" src="https://user-images.githubusercontent.com/12038224/78254238-e53cd480-74f5-11ea-848e-f9dfe9514925.png">